packages/client/src/features/select/select-feedback-action.ts

```diff
@@ -36,9 +36,25 @@ export namespace SelectFeedbackAction {
         return Action.hasKind(object, KIND) && hasArrayProp(object, 'selectedElementsIDs') && hasArrayProp(object, 'deselectedElementsIDs');
     }
 
-    export function create(options?: { selectedElementsIDs?: string[]; deselectedElementsIDs?: string[] }): SelectFeedbackAction {
+    export function create(options?: { selectedElementsIDs?: string[]; deselectedElementsIDs?: string[] | boolean }): SelectFeedbackAction {
         return { ...SelectAction.create(options), kind: KIND };
     }
+
+    export function addSelection(options: { selectedElementsIDs: string[] }): SelectFeedbackAction {
```

Comment:
I don't think that we need an options object here. We only use it in the create function because it's the default pattern to declare optional properties in an options object. Here we can just use plain arguments.

Reply:
I see, I thought it was to support spreading of additional options so I wanted to keep the pattern. But now I see what you mean and I changed it accordingly.